### DISCUSSION TITLE
Use Prisma in Redwoodjs tutorial

### DISCUSSION THREAD

**Discussion by kevin51jiang** [supabase/supabase#20436] [Contribute to Supabase]
# Improve documentation

## Link 

https://supabase.com/docs/guides/getting-started/tutorials/with-redwoodjs

## Describe the problem

From what I can understand, Prisma now supports cross-schema foreign keys, meaning that the tutorial should be able to use the built in Prisma DB.

https://www.prisma.io/docs/guides/database/multi-schema

## Describe the improvement

Rewrite/change the tutorial to use the Prisma that's built into Redwood JS

## Additional context

In the intro: 
![image](https://user-images.githubusercontent.com/33907581/216731201-b308af6f-ab14-486a-a3a2-f23a8a45b544.png)

**Comment by pratik-codes** [comment]
In this what should be done? should I remove this note or add that now the cross-schema foreign keys are now supported @kiwicopple
